### PR TITLE
Add service for GET on ingredients

### PR DESCRIPTION
**Pull request by nashirj**
also updated AddIngredientView to render/return ingredients

### CODE REVIEW

**Comment by nashirj** [comment]
This PR addresses #26 and partially addresses #6

**Comment by nashirj** [comment]
This PR also addresses #25

File: src/components/AddIngredientView.vue [hunk 1]
```diff
@@ -54,30 +87,62 @@ export default {
...
+        })
+        .then(
+          (response) => {
+            console.log(response);
+            this.searchResults = response;
+            this.errorString =
+              this.searchResults.length > 0 ? "" : "No search results found";
+          },
+          (error) => {
+            this.errorString =
```
Comment (mplekunov) [inline src/components/AddIngredientView.vue:138]: Is there any way to make it simpler?

Comment (nashirj) [reply src/components/AddIngredientView.vue]: Done.

**Review by mplekunov — APPROVED** [review]
LGTM 👍

**Comment by nashirj** [comment]
Thank you for the review

**Pull request merged**